ecommerce_integrations/unicommerce/customer.py

```diff
@@ -117,5 +117,6 @@ def _create_customer_address(uni_address, address_type, customer, also_shipping=
 			"links": [{"link_doctype": "Customer", "link_name": customer.name}],
 			"is_primary_address": int(address_type == "Billing"),
 			"is_shipping_address": int(also_shipping or address_type == "Shipping"),
+			"gst_category": 'Overseas' if country != 'India' else 'Unregistered'
```

Comment:
This shouldn't be assumed always that it's `Unregistered` if indian? 

Maybe leave this as customization on site?

Reply:
Removed